hnn_core/pyramidal.py

```diff
@@ -287,7 +298,7 @@ def _synapse_create(self, p_syn):
         self.synapses['apicaltuft_nmda'] = self.syn_create(
             self.dends['apical_tuft'](0.5), **p_syn['nmda'])
 
-        if self.name == 'L5Pyr':
+        if self.name == ('L5Pyr' or 'CustomL5Pyr'): #update for custom
```

Comment:
```suggestion
        if self.name == ('L5Pyr' or 'CustomL5Pyr'):  # update for custom
```
Our testing bots insist that you adhere to [PEP8](https://pep8.org). The easiest way to learn is to install `flake8` (using `pip` or `conda`) and run it on the `hnn_core`-folder.